src/helpers/testHelpers.ts

```diff
@@ -0,0 +1,45 @@
+import MongoMemoryServer from 'mongodb-memory-server-core';
+import mongoose from 'mongoose';
+
+export const setupTestDataBase = (databaseName: string) => {
+  const mongod = new MongoMemoryServer();
+
+  return {
+    beforeAll: async () => {
+      const uri = await mongod.getUri(databaseName);
+
+      await mongoose.connect(uri, {
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+        keepAlive: true,
+      });
+    },
+    afterAll: async () => {
+      await mongoose.connection.dropDatabase();
+      await mongoose.connection.close();
+      mongod.stop();
+    },
+  };
+};
+
+export const basicImportTest = (
+  importer,
+  testUrl: string,
+  createdSince?: Date,
+  createdUntil?: Date,
+) => {
+  return async () => {
+    const lt = await importer(testUrl, createdSince, createdUntil);
+    expect(lt).toBeDefined();
+    const json = lt?.toJSON();
+    delete json?._id;
```

Comment:
why is this deletion necessary?

Reply:
we need to delete the id from the db entry for the tests because it will be randomly generated and cause the snapshot to be different